src/main/webapp/instructorHelp.html

```diff
@@ -7461,7 +7568,7 @@ <h5 class="col-sm-2">
                                             Team 3
                                         </td>
                                         <td>
-                                            
+
```

Comment:
There are a lot of these changes in whitespaces for this file - is it possible to undo them?